cli/template/extras/src/app/page/with-auth-trpc-tw.tsx

```diff
@@ -1,10 +1,12 @@
+import { headers as dynamic } from "next/headers";
 import Link from "next/link";
 
 import { CreatePost } from "~/app/_components/create-post";
 import { getServerAuthSession } from "~/server/auth";
 import { api } from "~/trpc/server";
 
 export default async function Home() {
+  dynamic();
```

Comment:
no idea why it's not inferring this apge as dynamic from the trpc call (which calls cookies/headers etc..